website/sales/api/v2/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.urls import path
+
+from sales.api.v2.views import (
+    ShiftListView,
+    ShiftDetailView,
+    OrderListView,
+    OrderDetailView,
+)
+
+urlpatterns = [
+    path("sales/", ShiftListView.as_view(), name="shift-list"),
```

Comment:
Something with your url definitions makes it impossible for the docs schema to be generated.